.vscode/settings.json

```diff
@@ -1,14 +1,10 @@
 {
     "[python]": {
         "editor.tabSize": 4,
-        // Auto sort imports
-        "editor.codeActionsOnSave": {
-            "source.organizeImports": true
-        }
```

Comment:
Why remove these?

Reply:
Removed because they significantly slow down file save when doing a bulk find and replace. Since we have introduced lintrunner autofix, developers can use a single command for formatting which should be easy enough.